runtime/j9vm/java11vmi.c

```diff
@@ -843,6 +892,17 @@ JVM_DefineModule(JNIEnv * env, jobject module, jboolean isOpen, jstring version,
 			}
 		}
 	}
+#if JAVA_SPEC_VERSION >= 15
+cleanup:
+	if (NULL != packages) {
+		jsize pkgIndex = 0;
+		for (pkgIndex = 0; pkgIndex < numPackages; pkgIndex++) {
+			const char* packageName = packages[pkgIndex];
+			j9mem_free_memory((void *)packageName);
+		}
+		j9mem_free_memory((void *)packages);		
+	}
+#endif /* JAVA_SPEC_VERSION >= 15 */
 done:
```

Comment:
The `goto` on line 745 should target `cleanup` to avoid leaking `packages`.
That would be the last use of `done`; perhaps `cleanup` should just become `done`.

Reply:
Fixed.